packages/react/src/types/component-props.ts

```diff
@@ -102,41 +104,59 @@ export interface ReactFlowProps<NodeType extends Node = Node, EdgeType extends E
+  /** This event handler is called when mouse of a user leaves a node. */
   onNodeMouseLeave?: NodeMouseHandler<NodeType>;
-  /** This event handler is called when a user right clicks on a node */
+  /** This event handler is called when a user right-clicks on a node. */
   onNodeContextMenu?: NodeMouseHandler<NodeType>;
-  /** This event handler is called when a user starts to drag a node */
+  /** This event handler is called when a user starts to drag a node. */
   onNodeDragStart?: OnNodeDrag<NodeType>;
-  /** This event handler is called when a user drags a node */
+  /** This event handler is called when a user drags a node. */
   onNodeDrag?: OnNodeDrag<NodeType>;
-  /** This event handler is called when a user stops dragging a node */
+  /** This event handler is called when a user stops dragging a node. */
   onNodeDragStop?: OnNodeDrag<NodeType>;
-  /** This event handler is called when a user clicks on an edge */
+  /** This event handler is called when a user clicks on an edge. */
   onEdgeClick?: (event: ReactMouseEvent, edge: EdgeType) => void;
-  /** This event handler is called when a user right clicks on an edge */
+  /** This event handler is called when a user right-clicks on an edge. */
   onEdgeContextMenu?: EdgeMouseHandler<EdgeType>;
-  /** This event handler is called when mouse of a user enters an edge */
+  /** This event handler is called when mouse of a user enters an edge. */
   onEdgeMouseEnter?: EdgeMouseHandler<EdgeType>;
-  /** This event handler is called when mouse of a user moves over an edge */
+  /** This event handler is called when mouse of a user moves over an edge. */
   onEdgeMouseMove?: EdgeMouseHandler<EdgeType>;
-  /** This event handler is called when mouse of a user leaves an edge */
+  /** This event handler is called when mouse of a user leaves an edge. */
   onEdgeMouseLeave?: EdgeMouseHandler<EdgeType>;
-  /** This event handler is called when a user double clicks on an edge */
+  /** This event handler is called when a user double-clicks on an edge. */
   onEdgeDoubleClick?: EdgeMouseHandler<EdgeType>;
+  /**
+   * This handler is called when the source or target of a reconnectable edge is dragged from the
+   * current node. It will fire even if the edge's source or target do not end up changing.
+   *
+   * You can use the `reconnectEdge` utility to convert the connection to a new edge.
+   */
   onReconnect?: OnReconnect<EdgeType>;
+  /**
+   * This event fires when the user begins dragging the source or target of an editable edge.
+   */
   onReconnectStart?: (event: ReactMouseEvent, edge: EdgeType, handleType: HandleType) => void;
+  /**
+   * This event fires when the user releases the source or target of an editable edge. It is called
+   * even if an edge update does not occur.
+   *
+   * You can use the fourth `connectionState` parameter to have different behavior when a
+   * reconnection was unsuccessful.
+   */
+  // @TODO: connectionState outdated info?
```

Comment:
```suggestion
   */
```
Copied from reactflow api reference, I think the 4th parameter was removed?